src/components/rpc_client.rs

```diff
@@ -24,22 +24,21 @@ pub trait RpcClient {
 
 #[derive(Clone, Debug, Eq, PartialEq, serde::Serialize, serde::Deserialize)]
 pub struct GetBlock {
-        /// The hash of the requested block in hex.
-        hash: String,
+    /// The hash of the requested block in hex.
+    hash: String,
 
-        /// The number of confirmations of this block in the best chain,
-        /// or -1 if it is not in the best chain.
-        confirmations: i64,
+    /// The number of confirmations of this block in the best chain,
+    /// or -1 if it is not in the best chain.
+    confirmations: i64,
```

Comment:
consider `confirmations: Option<u64>` and `None` to mark "not in the best chain"

Reply:
This part is copied from Zebra RPC, I think it is a good idea to keep it untouched for consistency and future updates